pkg/kubectl/scale.go

```diff
@@ -139,6 +145,9 @@ func (scaler *ReplicationControllerScaler) ScaleSimple(namespace, name string, p
 	controller.Spec.Replicas = int(newSize)
 	// TODO: do retry on 409 errors here?
 	if _, err := scaler.c.UpdateReplicationController(namespace, controller); err != nil {
+		if errors.IsInvalid(err) {
```

Comment:
Quota rejections show up as a Forbidden error.  That said, admission control on quota does not care about the replica size for a ReplicationController.  It just blocks the ReplicationController from scaling up to the desired state by rejecting pods if it exceeds your allowed quota.